test/factories/classrooms.rb

```diff
@@ -4,6 +4,7 @@
   factory :classroom do
     sequence(:name) { |n| "Classroom #{n}" }
     grade { Classroom::GRADE_RANGE.sample }
+    trading_enabled { true }
```

Comment:
I'm happy that you take this shortcut! Because it highlights an issue.

Before this change, 
1) Admin/teacher needs to set up a classroom with students 
2) Then the student can trade active stock. 

After this change, 
1) Admin/teacher needs to set up a classroom with students
2) Then the admin/teacher needs to enable the trading, 
3) Then the student can trade active stock. 

The DB default is false, so it forces this extra step, and that's good.

If admins/teachers need to enable trading explicitly, tests should do so as well. Or at least, having two different factories for that to make the intent explicit. 

Something like create(:classroom, :with_trading_enabled)

Reply:
I agree this should default to false